src/ShallowTraversal.js

```diff
@@ -66,6 +69,18 @@ export function nodeHasId(node, id) {
   return propsOfNode(node).id === id;
 }
 
+
+export function nodeHasProperty(node, propKey, propValue) {
+  const nodeProps = propsOfNode(node);
+
+  if (propValue) {
+    return nodeProps[propKey] === propValue;
+  }
+
+  return nodeProps.hasOwnProperty(propKey);
```

Comment:
this is basically the `foo={false}` question. If it has a `foo` property but it's set to something falsy, do we still want to return true?